pages/jointhetribe.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import type { NextPage } from "next";
+import styles from "../styles/jointhetribe.module.css";
+import Button from "../components/UI/button";
+import { useAccount } from "@starknet-react/core";
+import { useDomainFromAddress } from "../hooks/naming";
+import FamousMembers from "../components/tribe/famousMembers";
+import TextCopy from "../components/UI/textCopy";
+import SearchMembers from "../components/tribe/searchMembers";
+
+const JoinTheTribe: NextPage = () => {
+  const { address } = useAccount();
+  const mainDomain = useDomainFromAddress(address ?? "");
+
+  return (
+    <div className={styles.page}>
+      <section className={styles.section1}>
+        <h1 className="title">Join the tribe</h1>
+        <div className={styles.centeredContainer}>
+          <p className="mr-5">Wear the stark signal</p>
+          <TextCopy text={mainDomain ? mainDomain.domain : "yourname.stark"} />
+        </div>
+        <FamousMembers />
+      </section>
+
+      <div className={styles.firstLeaf}>
+        <img alt="leaf" src="/leaves/new/leaf_02.svg" />
```

Comment:
Images file names for leaves start with an uppercase L at the start. They don't render on the vercel build